server/src/controllers/messages.ts

```diff
@@ -0,0 +1,16 @@
+export const USER_NOT_FOUND_ERROR_MESSAGE = "User not found!";
+export const USER_NAME_MAX_LENGTH = 100;
+export const RA_LENGTH = 8;
+export const MIN_PASSWORD_LENGTH = 6;
+export const EXISTING_RA_ERROR_MESSAGE = "We already have an account associated with that RA.";
+export const EXISTING_EMAIL_ERROR_MESSAGE = "We already have an account associated with that email address.";
+export const BIG_NAME_ERROR_MESSAGE = "Please shorten your name to 100 characters or less.";
+export const WRONG_RA_SIZE_ERROR_MESSAGE = "RA must be exactly 8 characters long.";
+export const INVALID_EMAIL_ERROR_MESSAGE = "Invalid email address.";
+export const INTERNAL_SERVER_ERROR_MESSAGE = "Internal Server ERROR! Contact the admin.";
+export const REQUIRED_ACCESS_GROUP_ERROR_MESSAGE = 'Access group is required!';
+export const ACCESS_GROUP_NOT_EXIST_ERROR_MESSAGE = 'Access group does not exist!';
+export const IMAGE_NOT_FOUND_ERROR_MESSAGE = 'No image found!';
+export const WRONG_PASSWORD_SIZE_ERROR_MESSAGE = 'Your password must be at least 6 characters long.';
+export const SUSCESSFUL_UPLOAD = 'Upload successful:';
+export const ERROR_UPLOADING_IMAGE = 'Error uploading image:';
```

Comment:
pra tudo que for constante, tira do `/controllers`. Pode criar uma pasta dentro de src chamada `/constants` e passa tudo pra lá